src/Carting.Api/Responses/V1/CartResponse.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace Carting.Api.Responses.V1
+{
+    public class CartResponse
+    {
+        /// <summary>
+        /// Identifier of the cart
+        /// </summary>
+		public string CartId { get; set; }
+        /// <summary>
+        /// List of cart items
+        /// </summary>
+		public IEnumerable<CartItem> CartItems { get; set; }
+    }
+
+    public class CartItem
+    {
+        /// <summary>
+        /// Id of the cart item
+        /// </summary>
+        public int _id { get; set; }
```

Comment:
Could you elaborate on name style here, why is completely different from other public properties?

Reply:
Fixed